src/CattyTests/Resources/ParserTests/Custom/ValidProjectAllBricks/ValidProjectAllBricks0991.xml

```diff
@@ -656,6 +656,15 @@
               <commentedOut>false</commentedOut>
               <spinnerSelection>80</spinnerSelection>
             </brick>
+            <brick type="SetBackgroundByIndexBrick">
```

Comment:
Please also add this Brick to `ValidProjectAllBricks0992.xml` which is the most recent version now.